api/src/test/java/org/openmrs/api/impl/ConditionServiceImplTest.java

```diff
@@ -61,21 +62,31 @@ public void setup (){
 	 */
 	@Test
 	public void saveCondition_shouldSaveNewCondition(){
+		
+		// Create Condition to test
+		String ns = "my ns";
+		String path = "my path";
 		Integer patientId = 2;
 		String uuid = "08002000-4469-12q3-551f-0339000c9a76";
 		CodedOrFreeText codedOrFreeText = new CodedOrFreeText();
 		Condition condition = new Condition();
+		condition.setFormField(ns, path);
```

Comment:
Modifying an existing test to set the form field creates an impression that it is required field. If not, i would rather create a new test.